stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -450,15 +449,15 @@ public class ChatClient internal constructor(
         channelId: String,
         offset: Int,
         limit: Int
-    ): Call<List<AttachmentWithDate>> =
+    ): Call<List<Attachment>> =
         getAttachments(channelType, channelId, offset, limit, ATTACHMENT_TYPE_FILE)
 
     public fun getImageAttachments(
         channelType: String,
         channelId: String,
         offset: Int,
```

Comment:
Maybe worth mentioning in the java doc that this is a message offset and not attachment offset